easybuild/framework/easyblock.py

```diff
@@ -214,6 +214,18 @@ def _init_log(self):
 
         self.log = fancylogger.getLogger(name=self.__class__.__name__, fname=False)
 
+        # deprecate self.log.error in easyblocks, self.log.fatal should be used instead
+        def error(msg, *args, **kwargs):
+            """DEPRECATED error() log method, use fatal() instead."""
+            self.log.deprecated("Use log.fatal() for logging/raise errors, rather than log.fatal()", '3.0')
+            new_msg = "EasyBuild crashed with an error %s: %s" % (self.caller_info(), msg % args)
```

Comment:
sorry, github cut the line and I couldn't see that last part -- yes the `%` is there